articles/active-directory/develop/msal-net-xamarin-android-considerations.md

```diff
@@ -107,10 +106,108 @@ Here's an example of a class that represents the values of the XML file:
+    <intent-filter>
+          <action android:name="android.intent.action.VIEW" />
+          <category android:name="android.intent.category.DEFAULT" />
+          <category android:name="android.intent.category.BROWSABLE" />
+          <data android:scheme="msauth"
+              android:host="Enter_the_Package_Name"
+              android:path="/Enter_the_Signature_Hash" />
+    </intent-filter>
+</activity>
+```
+
+Substitute the package name that you registered in the Azure portal for the `android:host=` value. Substitute the key hash that you registered in the Azure portal for the `android:path=` value. The signature hash should *not* be URL encoded. Ensure that a leading forward slash (`/`) appears at the beginning of your signature hash.
+
 ### Xamarin.Forms 4.3.x manifest
 
 Xamarin.Forms 4.3.x generates code that sets the `package` attribute to `com.companyname.{appName}` in *AndroidManifest.xml*. If you use `DataScheme` as `msal{client_id}`, then you might want to change the value to match the value of the `MainActivity.cs` namespace.
 
+## Android 11 support
+
+In order to utilize the system browser and brokered authentication on Android 11 you must first declare these packages so that they will be visible to the app.Apps targeting Android 10 (API 29) and below were able to query the OS for a list of which packages were available on the device at any given time. In the interest of privacy and security Android 11 is reducing the package visibility to a default list of OS packages and along with the packages that are specified in the app's AndoridManifest.xml file. 
+
+To enable the application to authenticate with both the system browser and broker, add the following to the *AndroidManifest.xml*:
+
+```xml
+<!-- Required for API Level 30 to make sure the app can detect browsers and other apps where communication is needed.-->
+<!--https://developer.android.com/training/basics/intents/package-visibility-use-cases-->
+<queries>
+  <package android:name="com.azure.authenticator" />
+  <package android:name="{Package Name}" />
+  <package android:name="com.microsoft.windowsintune.companyportal" />
+  <!-- Required for API Level 30 to make sure the app detect browsers
+      (that don't support custom tabs) -->
+  <intent>
+    <action android:name="android.intent.action.VIEW" />
+    <category android:name="android.intent.category.BROWSABLE" />
+    <data android:scheme="https" />
+  </intent>
+  <!-- Required for API Level 30 to make sure the app can detect browsers that support custom tabs -->
+  <!-- https://developers.google.com/web/updates/2020/07/custom-tabs-android-11#detecting_browsers_that_support_custom_tabs -->
+  <intent>
+    <action android:name="android.support.customtabs.action.CustomTabsService" />
+  </intent>
+</queries>
+``` 
+
+Replace `{Package Name}` with the application package name. 
+
+Your completed manifest that $DOES_THING_HERE should appear similar to the following:
```

Comment:
```suggestion
Your completed manifest that enables support for the system browser and brokered authentication should appear similar to the following:
```

Oops, sorry @trwalke, didn't mean for you to include the actual `$DOES_THING_HERE` token. My intent there was to be more specific about what the completed manifest does/now includes/etc.